### ISSUE TITLE
Conceptual problem

### ISSUE THREAD

**Issue by jjgonzalezplaza** [jtamames/SqueezeMeta#377]
Hi,

I wanted to ask a question regarding the whole process. I carried out an assembly with Seqmerge, and currently I am at the SQM analysis step (awesome milestone, to be honest). However, I did the following:

- Digital normalization of my reads (https://khmer-protocols.readthedocs.io/en/v0.8.4/metagenomics/2-diginorm.html).
It was the only way that I could carry out the assembly given my computing limitations, but I understand that what I see now is not that representative of the original sequencing.  Should I run again the whole process, but providing the already merged assembly as extassembly, this time with the non-normalized reads as input?
-The other thing is that I had three biological replicates per sample origin (e.g. 1.1, 1.2, 1.3) and I indicated in the samples file that 1.1_Left 1.1_Right (and the other two) belong to Sample1. In SQM I get only charts/tables for 1, 2, and 3 (not like your example of Hadza and Italian). Although I may be answering myself already, that means that I have to re-start the process this time indicating that each biological replicate is an unique sample (in sample file). Isn't it?
I was probably just thinking that I wanted a merged assembly of the 3 biological replicates, but I did not think on the downstream process.

**Comment by jjgonzalezplaza** [comment]
Hi,

I continued and finally got to load the files on R. I had several issues with the version of SQM tools, but was finally successful to install the latest one (only using the .tar.gz, not the .zip file). 

I then do this:

> library('SQMtools')
> NSt_ext = loadSQM('D:/T_Praga/Project_naph_st/Squeeze/NSt_ext')

Error: no se puede ubicar un vector de tamaño  62.5 Mb
Además: Warning message:
In sprintf("%s/results/tables/%s.superkingdom.%s.abund.tsv", project_path,  :
  one argument not used by format '%s/results/tables/%s.superkingdom.%s.abund.tsv'

Then if I try to load in SQMlite:

Error in loadSQMlite("D:/T_Praga/Project_naph_st/Squeeze/NSt_ext") : 
  Directory "D:/T_Praga/Project_naph_st/Squeeze/NSt_ext" does not seem to contain valid SqueezeMeta tables.

Not sure what I am doing wrong now. Otherwise I was trying to reproduce some of the charts in the SQM tutorial to analyze metagenomic data. Since I have generated the tsv files that were intended for SQM, all can be either parsed or used in other formats (although it takes more time than what I thought would be SQM).
